binding/python/util.py

```diff
@@ -125,13 +126,18 @@ def _pv_keyword_files_subdir():
 
     raise NotImplementedError('Unsupported platform')
 
-
-def pv_keyword_paths(relative):
+def pv_keyword_paths(relative, lang):
+    keyword_files_root = _append_by_lang('resources/keyword_files', lang)
     keyword_files_dir = \
-        os.path.join(os.path.dirname(__file__), relative, 'resources/keyword_files', _pv_keyword_files_subdir())
+        os.path.join(os.path.dirname(__file__), relative, keyword_files_root, _pv_keyword_files_subdir())
```

Comment:
same with this function, it is used to find the built-in keywords that we ship with the Porcupine Python SDK. We only package English keywords with the SDK, so the language isn't necessary. Use relative paths in the unit tests since those will always be run from the repo and can contain different languages.